BaseTools/Source/C/BrotliCompress/Makefile

```diff
@@ -26,12 +32,15 @@ ENC_OBJ = \
   brotli\c\enc\block_splitter.obj \
   brotli\c\enc\brotli_bit_stream.obj \
   brotli\c\enc\cluster.obj \
+  brotli\c\enc\command.obj \
+  brotli\c\enc\compound_dictionary.obj \
   brotli\c\enc\compress_fragment.obj \
   brotli\c\enc\compress_fragment_two_pass.obj \
   brotli\c\enc\dictionary_hash.obj \
   brotli\c\enc\encode.obj \
   brotli\c\enc\encoder_dict.obj \
   brotli\c\enc\entropy_encode.obj \
+  brotli\c\enc\fast_log.obj \

```

Comment:
I haven't actually tried building this on windows, so it would be good if someone (or CI) could confirm that this works